recipes/docker-binary/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "docker-binary" %}
```

Comment:
Should this just be called "docker"?

Reply:
Since the python interface is named `docker-py` that does free up `docker` to be used for the binary. I quite like `docker-binary` as being more explicit but don't have strong feelings so am happy to make the change if you want?

Reply:
Yeah, I think it makes more sense, because whether we build it or grab a pre-built binary shouldn't matter to the users of the package.  They'll want to install "docker"

Reply:
Whether you build the binary from source or simply repackage it what is being installed is a binary executable so I think it would work either way. Fine with changing it though so will try to get to that shortly...